src/climatevision/generator/refdata.py

```diff
@@ -622,6 +647,20 @@ def industry_dehst(self, ags: str):
         """TODO Function to read CO2e for each ags from DEHST Table."""
         return OptRow(self._industry_dehst, ags)
 
+    def traffic_air(self, ags: str):
+        """TODO"""
```

Comment:
TODO what?  Also do it ;-)

Reply:
Haha, einfach ein Copy-Paste von den anderen. Dachte, da muss noch jemand allgemein was machen, wenn das bei allen drin steht :D